<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="sys_tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QListWidget" name="sys_listWgt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="sys_code_le">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="sys_name_le">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="sys_upd_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>이름 변경</string>
     </property>
    </widget>
    <widget class="QToolButton" name="sys_ins_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>코드 추가</string>
     </property>
    </widget>
    <widget class="QToolButton" name="sys_del_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>코드 삭제</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gb_tab">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QToolButton" name="gb_ins_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>코드 추가</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gb_name_le">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="gb_del_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>230</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>코드 삭제</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gb_code_le">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="gb_upd_btn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>이름 변경</string>
     </property>
    </widget>
    <widget class="QListWidget" name="gb_listWgt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
